### DISCUSSION TITLE
No QT6 package found. Cannot build - Maya 2024

### DISCUSSION THREAD

**Discussion by voidreamer** [Autodesk/maya-hydra#128] [Q&A]
Hi, I'm trying to compile the latest release `0.6.1` for maya 2024.2 and Maya Usd 0.27 for ubuntu, but I get errors that it cannot find Qt6, is this release compatible with Qt5 ?
Error:
`No QT6 package found. Cannot build`

**Comment by bjorn-siegert-adsk** [comment]
Hi @voidreamer,
That's probably because of the Hydra Scene Browser which requires Qt (Maya 2025 moved to Qt6) but the scene browser is not mandatory and you should be able to build mayaHydra without it.
I would recommend building 0.6.1 against Maya 2025 though, as this is on a newer USD version which brings in a lot of improvements for Hydra's SceneIndex which we are utilizing.
